backend/sim/wallet/address.go

```diff
@@ -63,13 +62,13 @@ func (a *Address) Bytes() []byte {
 
 // ByteArray converts an address into a 64-byte array. The returned array
```

Comment:
byteArray....

Reply:
Not directly related to my changes, but shows that the linter only cares about exported functions...